kickstarter/android-oss
# NTV-334: Reward model

# 📲 What

- Migrated to kotlin
- Adopt @Parcelize 

# 🤔 Why
- Remove all references to AutoParcel library in Models, eventually we will be able to remove that dependency.

# 👀 See
- No user facing changes

|  |  |

# 📋 QA
- Pledge to a reward
- Pledge to a reward with addOns
- Check a few projects, that all the rewards with the reward items load correctly
- View your pledge, make sure the Reward description card is correct
- View your pledge with addOns, make sure the addOns description card is correct


# Story 📖

[NTV-334](https://kickstarter.atlassian.net/browse/NTV-334)
